src/main/java/core/basesyntax/EmployeeService.java

```diff
@@ -1,10 +1,25 @@
 package core.basesyntax;
 
+import java.util.Comparator;
 import java.util.List;
 import java.util.Set;
+import java.util.TreeSet;
 
 public class EmployeeService {
+    Comparator<Employee> employeeComparator = new Comparator<Employee>() {
+        @Override
+        public int compare(Employee employee1, Employee employee2) {
+            int comparedAges = Integer.compare(employee1.getAge(), employee2.getAge());
+            if (comparedAges != 0) {
+                return comparedAges;
+            }
+            return employee1.getName().compareTo(employee2.getName());
+        }
+    };
```

Comment:
this comparator's not part of the state of EmployeeServicem so let's not make this as class field especially considering the usage of the anonymous class